build.sbt

```diff
@@ -1,18 +1,19 @@
-lazy val akkaHttpVersion = "10.0.9"
-lazy val akkaVersion    = "2.5.3"
+lazy val akkaHttpVersion = "10.0.5"
```

Comment:
Yes, there is. I couldn't understand why it gave me certain errors while working through examples. I figured it might have something to do with different versions, and this was the way I managed to solve it.

See these remarks in gitter:

jluijs @jluijs Aug 18 10:11
probleem wat ik nu heb: could not find implicit value for parameter um: akka.http.scaladsl.unmarshalling.FromRequestUnmarshaller[battleship.routes.testJsonSupport.Test]
Ik moet dus iets met het unmarshallen en marshallen doen, maar ik krijg het al niet echt voor elkaar om de juiste imports te doen. Ik copy paste een voorbeeld, tik op alt-enter als intellij een rood keyword aangeeft, maar compiletime geeft hij dan zulk sootrt foutmeldingen:
object sprayjson is not a member of package akka.http.scaladsl.marshallers
[error] import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
ik probeer nu om json om te vormen zodat ik de variabelen kan gebruiken in een aanroep

jluijs @jluijs Aug 18 11:05
Heb wat zitten kutten met versies van akka-http, en nu compileert hij iig
mijn conclusie: Drie dingen zijn erg irritant: Hoe kom je erachter welke library je moet importeren, hoe kom je erachter welke librarydependecie in je build.sbt je moet toevoegen, en hoe kom je erachter welke versies van verschillende libraries en scala en akka met elkaar compatibel zijn.